whois-api/src/test/java/net/ripe/db/whois/api/rest/DomainObjectServiceIT.java

```diff
@@ -24,7 +25,7 @@
 import static org.junit.jupiter.api.Assertions.fail;
 
 @Tag("IntegrationTest")
-public class DomainObjectServiceTestIntegration extends AbstractIntegrationTest {
+public class DomainObjectServiceIT extends AbstractIntegrationTest {
```

Comment:
We usually follow a convention of putting IntegrationTest can you rename it to this?